scripts/speech_recognition/estimate_data_weights.py

```diff
@@ -0,0 +1,172 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from contextlib import contextmanager
+from typing import Sequence
+
+import click
+import numpy as np
+from omegaconf import DictConfig, ListConfig, OmegaConf
+
+from nemo.collections.common.data.lhotse.cutset import get_parser_fn
+
+
+@click.command()
+@click.argument("input_cfgs", type=click.Path(exists=True, dir_okay=False), nargs=-1)
+@click.argument("output_cfg", type=click.Path())
+@click.option(
+    "-t",
+    "--temperature",
+    type=float,
+    default=None,
+    multiple=True,
+    help="Temperature for re-weighting datasets. 1 is a neutral value. "
+    "Lower temperature over-samples smaller datasets, and vice versa. "
+    "Can be specified multiple times to apply a different temperature to each group level in the YAML config.",
+)
+@click.option(
+    "-s",
+    "--strategy",
+    type=click.Choice(["num_hours", "num_examples"]),
+    default="num_hours",
+    help="Strategy for choosing weights for each dataset.",
```

Comment:
Could you rephrase this? Maybe something along, `metric used for calculating dataset size`